docs/faq.rst

```diff
@@ -304,3 +304,36 @@ Run::
    conda install nb_conda
 
 Now all available conda environments (for example, one created by following the instructions `here <quickstart.rst#installation-using-conda>`_) will automatically be added to the list of kernels.
+
+
+How do I cite Parsl?
+^^^^^^^^^^^^^^^^^^^^
+
+To cite Parsl in publications, please use the following:
+
+Babuji, Y., Woodard, A., Li, Z., Katz, D. S., Clifford, B., Kumar, R., Lacinski, L., Chard, R., Wozniak, J., Foster, I., Wilde, M., and Chard, K., Parsl: Pervasive Parallel Programming in Python. 28th ACM International Symposium on High-Performance Parallel and Distributed Computing (HPDC). 2019.
```

Comment:
Should add "https://doi.org/10.1145/3307681.3325400" after the year.

Reply:
Fixed.